web/core/components/issues/issue-layouts/kanban/swimlanes.tsx

```diff
@@ -21,6 +21,7 @@ import { getGroupByColumns, isWorkspaceLevel, GroupDropLocation } from "../utils
 import { KanBan } from "./default";
 import { HeaderGroupByCard } from "./headers/group-by-card";
 import { HeaderSubGroupByCard } from "./headers/sub-group-by-card";
+import { Row } from "@plane/ui";
```

Comment:
Can you please run `yarn lint`, to fix the lint errors. This should have thrown a lint error